packages/assistant-stream/src/core/modules/assistant-stream.ts

```
@@ -0,0 +1,230 @@
+    return this._merger.readable;
+  }
+
+  private _closeSubscriber: undefined | (() => void);
+  __internal_subscribeToClose(callback: () => void) {
+    this._closeSubscriber = callback;
+  }
+
+  private _closeSubscriber: undefined | (() => void);
+  __internal_subscribeToClose(callback: () => void) {
+    this._closeSubscriber = callback;
+  }
```

Biome (1.9.4)

[error] Duplicate class member name "__internal_subscribeToClose"



(lint/suspicious/noDuplicateClassMembers)